src/components/buttons/GroupOrder.js

```diff
@@ -26,10 +36,13 @@ const GroupOrder = () => {
     dispatch(openModal({ type: 'groupOrder', args: { reviewOrders } }))
   }
 
+  const locationType = useSelector(selectLocationType)
+  console.log(locationType)
```

Comment:
@franciscoxrra I won't hold up the merge for this, but it would be good to get in the habit of clearing out all eslint warnings before creating a PR.

Reply:
sorry, my bad. I made "unused vars" an error instead of a warning now. Maybe we should use the "no-console" warnings and make exceptions for the serviceWorker 🤔